### PR TITLE
SAMZA-1232: Log configuration value in RunLoopFactory

### PR DESCRIPTION
**Pull request by shanthoosh**

### CODE REVIEW

**Comment by shanthoosh** [comment]
@xinyuiscool Please take a look when you get a chance.

File: samza-core/src/main/java/org/apache/samza/container/RunLoopFactory.java [hunk 1]
```diff
@@ -87,9 +87,11 @@ public Boolean apply(TaskInstance t) {
     } else {
       Integer taskMaxConcurrency = config.getMaxConcurrency().getOrElse(defaultValue(1));
 
+      log.info("Got max messages in flight: " + taskMaxConcurrency);
+
       boolean isAsyncCommitEnabled = config.getAsyncCommit().getOrElse(defaultValue(false));
 
-      log.info("Got max messages in flight: " + taskMaxConcurrency);
```
Comment (vjagadish1989) [inline samza-core/src/main/java/org/apache/samza/container/RunLoopFactory.java]: Nuke "from config" from the log line. I'm guessing it is implicit?

**Review by vjagadish1989 — APPROVED** [review]
approved. thank you Shanthosh for the contribution. looks good

File: samza-core/src/main/java/org/apache/samza/container/RunLoopFactory.java (hunk 1 above)
Comment (shanthoosh) [inline samza-core/src/main/java/org/apache/samza/container/RunLoopFactory.java]: Fixed.

File: samza-core/src/main/java/org/apache/samza/container/RunLoopFactory.java [hunk 2]
```diff
@@ -87,9 +87,11 @@ public Boolean apply(TaskInstance t) {
     } else {
       Integer taskMaxConcurrency = config.getMaxConcurrency().getOrElse(defaultValue(1));
 
+      log.info("Got max messages in flight: " + taskMaxConcurrency);
```
Comment (sborya) [inline samza-core/src/main/java/org/apache/samza/container/RunLoopFactory.java]: Not a big deal, but I'de add the word taskMaxConcurrency={}, to tie it better to the code. By just reading log 'messages in flight' might have different connotations.

**Comment by prateekm** [comment]
@shanthoosh is this ready to merge?

File: samza-core/src/main/java/org/apache/samza/container/RunLoopFactory.java (hunk 2 above)
Comment (shanthoosh) [inline samza-core/src/main/java/org/apache/samza/container/RunLoopFactory.java]: @sborya    
Fixed

Changed other log statements of this form for consistency within this file.

**Pull request closed without merging**